docs/web/user_guide.md

```diff
@@ -1579,6 +1583,49 @@ not found, the user will be asked, in the command-line, to provide credentials.
 CodeChecker parse ./my_plists --suppress generated.suppress --export-source-suppress
 ```
 
+### Export comments and review statuses (`export`) <a name="cmd-export"></a>
+
+```sh
+usage: CodeChecker cmd export [-h] [-n RUN_NAME [RUN_NAME ...]] [--url PRODUCT_URL] [--verbose {info,debug_analyzer,debug}]
+
+Export data (comments, review statuses) from a running CodeChecker server into a json format
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -n RUN_NAME [RUN_NAME ...], --name RUN_NAME [RUN_NAME ...]
+                        Name of the analysis run.
+
+common arguments:
+  --url PRODUCT_URL     The URL of the product which will be accessed by the client, in the format of '[http[s]://]host:port/Endpoint'. (default:
+                        localhost:8001/Default)
+  --verbose {info,debug_analyzer,debug}
+                        Set verbosity level.
```

Comment:
We do not need to list `common arguments` here. We don't list it for the other commands too.

Also please don't remove line breaks. Just run this command `CodeChecker cmd export -h` and copy-paste the output to here without the common arguments section :blush: 

```suggestion
usage: CodeChecker cmd export [-h] [-n RUN_NAME [RUN_NAME ...]]
                              [--url PRODUCT_URL]
                              [--verbose {info,debug_analyzer,debug}]

Export data (comments, review statuses) from a running CodeChecker server into
a json format

optional arguments:
  -h, --help            show this help message and exit
  -n RUN_NAME [RUN_NAME ...], --name RUN_NAME [RUN_NAME ...]
                        Name of the analysis run.
```